_includes/footer.html

```diff
@@ -15,7 +15,7 @@
           <a href="https://www.linkedin.com/company/esteemed/" class="footerbottom__social-icon"><img src="/img/linkedin.svg"><span class="show-for-sr">Linkedin</span></a>
           <a href="mailto:{{ site.email }}" class="footerbottom__social-icon" target="_blank"><img src="/img/email.svg"><span class="show-for-sr">Email</span></a>
         </div>
-        <div class="footerbottom__copyright">Copyright© 2019 Esteemed. <a href="/privacy-policy">Privacy</a> | <a href="/terms-use">Terms</a></div>
+        <div class="footerbottom__copyright">Copyright© 2020 Esteemed Inc. <a href="/privacy-policy">Privacy</a> | <a href="/terms-use">Terms</a></div>
```

Comment:
As noted in Slack, please utilize the liquid date tag here instead.

https://shopify.github.io/liquid/filters/date/

So, for this, it'd be `{{ "now" | date: "%Y" }}`